cost-analyzer/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: development
 description: Kubecost Helm chart - monitor your cloud costs!
 name: cost-analyzer
-version: 2.3.3
+version: 0.0.0-dev  # This is a placeholder used on the develop branch. If installing Kubecost, please reference a release branch.
```

Comment:
This field requires semVer, which is why I used `0.0.0-dev`. Open to any better suggestions!